packages/multichain-network-controller/src/constants.ts

```diff
@@ -0,0 +1,45 @@
+import { NetworkStatus } from '@metamask/network-controller';
+
+import type {
+  MultichainNetworkConfiguration,
+  MultichainNetworkMetadata,
+} from './MultichainNetworkController';
+
+export const bitcoinCaip2ChainId = 'bip122:000000000019d6689c085ae165831e93';
+export const solanaCaip2ChainId = 'solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp';
+
+export const multichainNetworkConfigurations: Record<string, MultichainNetworkConfiguration> = {
```

Comment:
Where are these constants used?

Reply:
I think they were added here to import in the consumer (mobile/extension) and when we initialise the controller we can have this source of truth of data for non evm networks!
Do you think since it's not directly used in the multichain network controller doesn't make sense?

Reply:
Ah. For NetworkController we prepopulate the state with network configurations for all known supported Infura networks, so that a consumer can access information for each network without having to explicitly add it:

https://github.com/MetaMask/core/blob/846253ae2a2c00999a34bf73e9af43b8799822ca/packages/network-controller/src/NetworkController.ts#L542

Does it make sense to do a similar thing for the MultichainNetworkController? I know things work differently here so I'm not sure.